src/store.js

```diff
@@ -11,6 +11,13 @@ export class OverriddenComponentRepository {
     this.components[id] = Component;
   };
 
+  append = (id, Component) => {
+    if (!Array.isArray(this.components[id])) {
```

Comment:
Should this fail if there's already a single-component override? Or automatically convert the existing entry to an array containing that one?

Right now `add` followed by `append` silently discards the initial element.